src/openms/source/ANALYSIS/OPENSWATH/OpenSwathScoring.cpp

```diff
@@ -257,9 +258,20 @@ namespace OpenMS
 
       if (compound.isPeptide())
       {
-        diascoring.dia_ms1_isotope_scores(precursor_mz, ms1_spectrum,
-                                          precursor_charge, scores.ms1_isotope_correlation,
```

Comment:
By the way, why was nothing complaining here? dia_ms1_isotope_scores takes a size_t as charge. If precursor_charge is negative (from the transition) you probably get quite some nonsense.